addons/payment-stripe.php

```diff
@@ -408,7 +413,7 @@ protected function charge( $order, $payment_token, $token_id, $receipt_email ) {
 		$statement_descriptor = $camptix->substr_bytes( strip_tags( $this->camptix_options['event_name'] ), 0, 22 );
 
 		$request_args = array(
-			'user-agent' => 'CampTix-Stripe/' . CampTix_Stripe::VERSION . ' (https://github.com/dd32/CampTix-Stripe-Payment-Gateway)',
+			'user-agent' => 'CampTix-Stripe/' . 'addon-test' . ' (https://github.com/dd32/CampTix-Stripe-Payment-Gateway)',
```

Comment:
One idea for a user-agent string here would be something simple like `'CampTix/' . $this->version`

Reply:
yes, this seems good.